.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        rust_versions: ["stable", "1.69"]
```

Comment:
My goal is to support at least a year old compiler, why did we need to bump this?

Reply:
Toml has been bumping things up fairly quickly. I set a max version on them and we should be good going forward. 1.69 is quite a year old (4/1/23) but we can stick here until the June time frame